ui/tests/dashboard.spec.ts

```diff
@@ -57,45 +69,45 @@ describe('dashboard', function () {
         const apiKey = process.env.BITTREX_API_KEY as string;
         const apiSecret = process.env.BITTREX_API_SECRET as string;
 
-        before(async () => {
-            await retry( async() => {
-                navigateTo(client,  '#user_settings_button');
-            });
-            await closeAddYourSettingsPopup(client);
+        before(async function () {
+            await setupTest(app, `${title}_after_adding`, async () => {
+                await navigateTo(client, '#user_settings_button');
+                await closeAddYourSettingsPopup(client);
 
-            await client.waitForVisible('#blockchain_balances_panel_body', METHOD_TIMEOUT);
+                await client.waitForVisible('#blockchain_balances_panel_body', METHOD_TIMEOUT);
 
-            await controller.addAccount(AccountType.ETH, ethAccount);
-            await controller.addAccount(AccountType.BTC, btcAccount);
+                await controller.addAccount(AccountType.ETH, ethAccount);
+                await controller.addAccount(AccountType.BTC, btcAccount);
 
-            await client.waitForExist('#btcchain_per_account_table', METHOD_TIMEOUT);
+                await client.waitForExist('#btcchain_per_account_table', METHOD_TIMEOUT);
 
-            await controller.addFiatValue();
-            await controller.addExchange(apiKey, apiSecret);
+                await controller.addFiatValue();
+                await controller.addExchange(apiKey, apiSecret);
 
-            await navigateToDashboard(client);
+                await navigateToDashboard(client);
```

Comment:
What are these whitespace changes?

Reply:
So these whitespace changes here are because the setup code has been moved into an anonymous function as part of the `setupTest` function. It seems that during the merge I missed the retry part on the navigate though.